dev/appveyor-install-dependencies.ps1

```diff
@@ -26,7 +26,7 @@ Function InstallR {
   }
 
   $urlPath = ""
-  $latestVer = $(ConvertFrom-JSON $(Invoke-WebRequest http://rversions.r-pkg.org/r-release).Content).version
+  $latestVer = $(ConvertFrom-JSON $(Invoke-WebRequest http://rversions.r-pkg.org/r-release-win).Content).version
```

Comment:
Yes, but I think it'd be a good idea if we introduce something like `latest` as discussed in #15709 later.